meta-mbl-apps/recipes-connectivity/mbl-cloud-client/mbl-cloud-client.bb

```diff
@@ -123,14 +128,14 @@ do_compile() {
 
 do_install() {
     install -d "${D}/opt/arm"
-    install "${S}/cloud-services/mbl-cloud-client/__${TARGET}/${RELEASE_TYPE}/mbl-cloud-client" "${D}/opt/arm"
-    install "${S}/cloud-services/mbl-cloud-client/__${TARGET}/${RELEASE_TYPE}/pelion-provisioning-util" "${D}/opt/arm"
+    output_dir="${S}/cloud-services/mbl-cloud-client/__${TARGET}"
+    install "${output_dir}/${RELEASE_TYPE}/mbl-cloud-client" "${D}/opt/arm"
```

Comment:
Can you please enforce the file permissions on the installation? I suppose you want `755`.

Reply:
7d62cd637a7bd35c3903a9b8f32cd9c41e0b880d